arm-notificationhubs/2016-03-01/swagger/notificationhubs.json

```diff
@@ -0,0 +1,1742 @@
+                    "description": "Gets or sets name"
+                },
+                "type": {
+                    "type": "string",
+                    "description": "Gets or sets resource type"
+                },
+                "tags": {
+                    "type": "object",
+                    "additionalProperties": {
+                        "type": "string"
+                    },
+                    "description": "Gets or sets tags"
+                },
+                "isAvailiable": {
+                    "type": "boolean",
+                    "description": "Gets or sets true if the name is available and can be used to create new Namespace/NotificationHub. Otherwise false."
+                }
+            },
+            "description": "Description of a CheckAvailibility resource."
+        },
+        "NamespaceProperties": {
+            "properties": {
+                "name": {
+                    "type": "string",
+                    "description": "The name of the namespace."
+                },
+                "provisioningState": {
+                    "type": "string",
+                    "description": "Gets or sets provisioning state of the Namespace."
+                },
+                "region": {
+                    "externalDocs": {
+                        "url": "http://msdn.microsoft.com/en-us/library/windowsazure/jj873988.aspx"
+                    },
+                    "type": "string",
+                    "description": "Specifies the targeted region in which the namespace should be created. It can be any of the following values: Australia EastAustralia SoutheastCentral USEast USEast US 2West USNorth Central USSouth Central USEast AsiaSoutheast AsiaBrazil SouthJapan EastJapan WestNorth EuropeWest Europe"
+                },
+                "status": {
+                    "externalDocs": {
+                        "url": "http://msdn.microsoft.com/en-us/library/windowsazure/jj873988.aspx"
+                    },
+                    "type": "string",
+                    "description": "Status of the namespace. It can be any of these values:1 = Created/Active2 = Creating3 = Suspended4 = Deleting"
+                },
+                "createdAt": {
+                    "type": "string",
+                    "format": "date-time",
+                    "description": "The time the namespace was created."
+                },
+                "serviceBusEndpoint": {
+                    "externalDocs": {
+                        "url": "http://msdn.microsoft.com/en-us/library/windowsazure/jj873988.aspx"
+                    },
+                    "type": "string",
+                    "description": "Endpoint you can use to perform NotificationHub operations."
+                },
+                "subscriptionId": {
+                    "type": "string",
+                    "description": "The Id of the Azure subscription associated with the namespace."
```

Comment:
servicebus subscription and azure subscriptionId are two very different things. Hence making sure if this is the right thing.